docs/tempo/website/configuration/ingestion-limit.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Ingestion limits
+---
+
+# Ingestion limits
+
+The default per user or per tenant trace limits in Tempo may not be sufficient in high volume tracing environments. The following message, for example, indicates that your tenants are exceeding the trace limits imposed on them.
+ ```
+    max live traces per tenant exceeded: per-user traces limit (local: 10000 global: 0 actual local: 10000) exceeded
+```    
+
+In such environments, increase the limits either individually by tenants or globally across the cluster
+so that your traces are not limited by the default settings.
+
+This following sections describe the parameters whose limits can be overridden, as well as the methods of
+overriding the default limits.
+
+## Override parameters
+
+You can override the default values of the following parameters:
+
+   - `ingestion_burst_size` : Burst size used in span ingestion. Default is `100,000`.
+   - `ingestion_rate_limit` : Per-user ingestion rate limit in spans per second. Default is `100,000`.
+   - `max_spans_per_trace` : Maximum number of spans per trace.  `0` to disable. Default is `50,000`.
+   - `max_traces_per_user`: Maximum number of active traces per user, per ingester. `0` to disable. Default is `10,000`.
+
+Both the `ingestion_burst_size` and `ingestion_rate_limit` parameters control the rate limit. When these limits exceed the following message is logged:
+
+```
+    RATE_LIMITED: ingestion rate limit (100000 spans) exceeded while adding 10 spans
+```    
+
+When the limit for the `max_spans_per_trace` parameter exceeds the following message is logged:
+
+```
+    TRACE_TOO_LARGE: totalSpans (50000) exceeded while adding 2 spans
+```
+
+Finally, when the limit for the `max_traces_per_user` parameter exceeds the following message is logged:
+
+```
+LIVE_TRACES_EXCEEDED: max live traces per tenant exceeded: per-user traces limit (local: 10000 global: 0 actual local: 1) exceeded
+```
+## Specify overrides to the default settings
```

Comment:
Combine the sections "Specify overrides to the default settings" and "Standard overrides".  These are basically the same thing: Telling the user how to configure non-default limits.